application/server/server.go

```diff
@@ -367,6 +372,26 @@ func logError(err error, method string) {
 	}
 }
 
+func textDocumentDidOpenHandler() jrpc2.Handler {
+	return handler.New(func(_ context.Context, params sglsp.DidOpenTextDocumentParams) (any, error) {
+		filePath := uri.PathFromUri(params.TextDocument.URI)
+		logger := log.With().Str("method", "TextDocumentDidOpenHandler").Str("documentURI", filePath).Logger()
+
+		logger.Info().Msg("Receiving")
+		folder := workspace.Get().GetFolderContaining(filePath)
+		issues := folder.DocumentDiagnosticsFromCache(filePath)
```

Comment:
This doesn't respect severity filtering.

Reply:
Ah yeah, good call.

Reply:
oops. not fixed yet.